TEST.js

```diff
@@ -1,183 +0,0 @@
-//readline library link
-const readline = require("readline");
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-start();
-//Variable Declarations
-let maxRange;
-let minRange;
-let numGuess;
-let compPick = "";
-
-//Game Start
-async function start() {
-  console.log(
-    "Let's play a game where you (Big BAUSE) make up a number and I (BeepBoopMachine) try to guess it. \n"
-  );
-  setInitRange();
-}
-
-//Set initial range with stops to make sure input is a number
-async function setInitRange() {
-  maxRange = await ask("So what's the highest number I can guess?\n");
-  while (isNaN(maxRange)) {
-    maxRange = await ask("Let's try this again. Please enter a number. \n");
-  }
-  maxRange = Number(maxRange);
-
-  minRange = await ask("And what is the lowest number I can guess?\n");
-  while (isNaN(minRange)) {
-    minRange = await ask("Let's try this again. Please enter a number.\n");
-  }
-  minRange = Number(minRange);
```

Comment:
The use of `Number()` here is assuring that minRange is indeed a number (and not a string for example), while in the `Human Guesses` function further down it appears that `Number()` is being used to try and pick a random number within a range which is where the bug in your second program appears to be originating from.